### PR TITLE
{bio}[iccifort/2020.1.217] fastp v0.20.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3114.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/076858ed4698cefa2957e0bb347cd520 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/fec8400939ee06b4cdefd335a98d37ed for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4cfe02e52a44519b8b86d010234ee69e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node126.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/6c3627ac91db666aa2369b6c456f137e for a full test report.

File: easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+homepage = 'https://github.com/OpenGene/fastp'
+description = """A tool designed to provide fast all-in-one preprocessing for FastQ files.
+ This tool is developed in C++ with multithreading supported to afford high performance."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
+
+source_urls = ['https://github.com/OpenGene/fastp/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e1b663717850bed7cb560c0c540e9a05dd9448ec76978faaf853a6959fd5b1b3']
+
```
Comment (lexming) [inline easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb:15]: There is a missing dependency on `zlib`
```suggestion

dependencies = [
    ('zlib', '1.2.11'),
]

```

File: easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+buildopts = ' CXX=${CXX}'
+
+preinstallopts = 'mkdir -p %(installdir)s/bin && '
+
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'dirs': [],
+    'files': ['bin/fastp'],
```
Comment (lexming) [inline easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb:26]: Blasphemy! `files` always goes before `dirs`
```suggestion
    'files': ['bin/fastp'],
    'dirs': [],
```

File: easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+name = 'fastp'
+version = '0.20.1'
+
+homepage = 'https://github.com/OpenGene/fastp'
+description = """A tool designed to provide fast all-in-one preprocessing for FastQ files.
+ This tool is developed in C++ with multithreading supported to afford high performance."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
+
+source_urls = ['https://github.com/OpenGene/fastp/archive/']
```
Comment (lexming) [inline easybuild/easyconfigs/f/fastp/fastp-0.20.1-iccifort-2020.1.217.eb:12]: Just for consistency with the previous easyconfig of `fastp`
```suggestion
# https://github.com/OpenGene/fastp
github_account = 'OpenGene'
source_urls = [GITHUB_SOURCE]
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
@boegel ping on the requested changes?

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**